<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="txtMongoUri">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>MongoURI</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtDatabaseName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Database Name</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtCollectionName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Collection Name</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtFileName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>File name with path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Info</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
